.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,56 @@
+<!-- Thanks for filing an OSCC issue! You're making our world a better place!-->
+
+<!-- Questions about purchasing and pricing for OSCC boards or DriveKit?
+Shoot an email to drivekit@polysync.io -->
+
+
+
+<!--
+Interested in a new feature?
```

Comment:
Given that we're dealing with both software and hardware, I expect that we'll probably get as many (if not more) bug reports than we will receive feature requests. Can we modify this template to take that into consideration?